homeassistant/components/blebox/sensor.py

```diff
@@ -1,11 +1,13 @@
 """BleBox sensor entities."""
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
-from .const import BLEBOX_TO_HASS_DEVICE_CLASSES, BLEBOX_TO_UNIT_MAP
+from .const import BLEBOX_TO_UNIT_MAP
```

Comment:
Maybe also move `BLEBOX_TO_UNIT_MAP` to `sensor.py` if it is not used by other platforms